module3/projects/index.md

```diff
@@ -3,3 +3,17 @@ layout: page
 title: Module 3 - Projects
 ---
 
+## Required Projects
+
+The following projects will be assigned and their completion is required to be promoted to module 2.  
+The project specs will be linked below as each project is assigned.
+
+- Week 1-2 (BE Solo): [Viewing Party](./viewing_party)
+- Week 3 (Paired): [FE Project](./)
+- Week 4-5 (Group): [Consultancy](./consultancy/)
+- Week 6 (Solo Final Project): [Mock Take Home Challenge](./take_home)
```

Comment:
As I mentioned - I'd opt for this to be hidden on day 1 of the inning, so the inning starts with just the first project and consultancy showing (like current BE mod 3)

Reply:
Another option would be to have the general outline/spec public the whole time, but we share the actual task (deets on tea time) the morning of kickoff. Eventually, we might have a couple of tasks we pull from, so we could keep those pages hidden (they have a URL but no direct link on our site) and just share the URL in Slack that morning.

Reply:
Yeah my proposition was to hide only the prompt so that the rubric and general outline of what is expected will be available the whole time. The midmod will use the same rubric so hiding just the actual technical prompt seems to make sense here.